bedrock/firefox/templates/firefox/firstrun/firstrun_quantum.html

```diff
@@ -0,0 +1,78 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% add_lang_files "firefox/new/onboarding" %}
```

Comment:
We shouldn't use this file, it's already been replaced by `firefox/new/quantum` and scheduled for removal.